.prettierrc

```
@@ -1,7 +1,3 @@
 {
-    "tabWidth": 4,
-    "trailingComma": "all",
-    "singleQuote": true,
-    "bracketSpacing": true,
-    "semi": false
-}
+    "extends": "@guardian/prettier"
```

i think you just want this
```suggestion
"@guardian/prettier"
```

---

Thanks @sndrs . We're seeing a lot more errors in VSCode now but the CLI seems to ignore a lot of them - is this to be expected? I wondered if we need to be more implicit in saying which exact folders to cover?

Would be good to find a way of mass formatting that doesn't involve opening each file individually and saving it.

---

Ignore this, we've resolved it now. We were missing some quotation marks in the path for the lint script.